Formula/docksal.rb

```diff
@@ -0,0 +1,25 @@
+class Docksal < Formula
+  desc "Docker powered web development environments for macOS, Windows & Linux"
+  homepage "https://docksal.io"
+  url "https://github.com/docksal/docksal/archive/v1.12.3.tar.gz"
+  sha256 "252a8ce0194fbcacb0a2b7dd6d578843f4ada2cd31c1859350a53791827932a0"
+
+  def install
+    mkdir_p "~/.docksal"
+    touch "~/.docksal/docksal.env"
+    system "echo", "'DOCKER_NATIVE=1'", ">>", "~/.docksal/docksal.env"
+    bin.install "bin/fin"
+  end
+
+  def caveats
+    s = "VirtualBox can be installed via `brew cask install virtualbox`\n"
```

Comment:
`caveats` is not for generic documentation, but for Homebrew-specific advice, when we ship with non-default configuration.